### DISCUSSION TITLE
Setting number of columns

### DISCUSSION THREAD

**Discussion by dalgas98** [wilgaboury/super-speedy-dial#77] [Ideas]
I'd love to have the ability to set the number of columns.  Hope it is on the todo list.

**Comment by wilgaboury** [comment]
Thank you for the idea. I appreciate getting feedback about ways to improve the plugin, and will add this feature to the todo list. BUT, to be very honest, I'm not sure I will ever get around to implementing it.

The problem is that it adds a lot of code and configuration complexity (things I'd like to avoid). For instance, what happens when the user resizes the screen? Do the tiles stay a set size and the window becomes horizontally scrollable? Does the entire tile view scale with the window size? Does the text also scale and become smaller and less readable? If these behaviors are configurable how can they be easily understood from the settings interface?

I think it would be a nice feature to add, but it will take a lot of thought, planning and effort to implement. And currently, there are a number of other features/issues that are higher on my priority list.

If it's any condolence, the height, width, and gap between tiles is fully configurable, and changing them will effectively change the number of columns.
